huxley/www/static/js/huxley/components/RegistrationView.js

```diff
@@ -571,6 +541,15 @@ var RegistrationView = React.createClass({
     this.setState(change);
   },
 
+   _handleCommitteePreferenceChange: function(committee) {
+    var index = this.state.committee_prefs.indexOf(committee.id);
+    if (index < 0) {
+      this.state.committee_prefs.push(committee.id);
+    } else {
+      this.state.committee_prefs.splice(index, 1);
```

Comment:
Similarly, here we'd use a `filter()` call:
```js
this.setState({
  committee_prefs: this.state.committee_prefs.filter(function(id) {
    return committee.id !== id;
  }),
});
```
Filtering creates a new array, and the old one isn't modified.